cypress/fixtures/json/bk_rcapi_template_default.json

```diff
@@ -1,28 +1,28 @@
 {
-    "template": [
-        {
-            "uri": "https://api.ramanchada.ideaconsult.net/template/26475cb3-a4e8-4b25-844a-87e3b244d73c",
-            "uuid": "26475cb3-a4e8-4b25-844a-87e3b244d73c",
-            "METHOD": "Dose response  - change with your assay",
-            "timestamp": "2024-12-05T16:09:30.677691",
-            "PROTOCOL_CATEGORY_CODE": "ENM_0000068_SECTION",
-            "EXPERIMENT": "Generic dose response template",
-            "template_name": "Dose response template with plate layout",
-            "template_status": "FINALIZED",
-            "template_author": " NJ",
-            "template_acknowledgment": "enanomapper"
-        },
-        {
-            "uri": "https://api.ramanchada.ideaconsult.net/template/8eb6a363-d1c3-4380-8562-2651db4d227b",
-            "uuid": "8eb6a363-d1c3-4380-8562-2651db4d227b",
-            "METHOD": "Raman spectrum",
-            "timestamp": "2024-06-25T12:50:34.656448",
-            "PROTOCOL_CATEGORY_CODE": "ANALYTICAL_METHODS_SECTION",
-            "EXPERIMENT": "CHARISMA ",
-            "template_name": "Raman spectra",
-            "template_status": "DRAFT",
-            "template_author": "Nina Jeliazkova",
-            "template_acknowledgment": "CHARISMA"
-        }
-    ]
+  "template": [
+    {
+      "uri": "https://api.ramanchada.ideaconsult.net/template/26475cb3-a4e8-4b25-844a-87e3b244d73c",
+      "uuid": "26475cb3-a4e8-4b25-844a-87e3b244d73c",
+      "METHOD": "Dose response  - change with your assay",
+      "timestamp": "2024-12-05T16:09:30.677691",
+      "PROTOCOL_CATEGORY_CODE": "ENM_0000068_SECTION",
+      "EXPERIMENT": "Generic dose response template",
+      "template_name": "Dose response template with plate layout",
+      "template_status": "FINALIZED",
+      "template_author": " NJ",
+      "template_acknowledgment": "enanomapper"
+    },
+    {
+      "uri": "https://api-test.ramanchada.ideaconsult.net/template/b2903810-718e-43e2-acfa-f70666ae078b",
+      "uuid": "b2903810-718e-43e2-acfa-f70666ae078b",
+      "METHOD": null,
+      "timestamp": "2024-12-05T09:23:39.069850",
+      "PROTOCOL_CATEGORY_CODE": "?",
+      "EXPERIMENT": "?",
+      "template_name": "Cypress Test",
+      "template_status": "DRAFT",
+      "template_author": "Sergey",
+      "template_acknowledgment": "Test"
+    }
+  ]
```

Comment:
Isn't it better to keep a real life example here? I think in the future we might also want to add different examples representing different edge cases. But I'm not sure I understand what is the advantage of this change.

Reply:
@sergesoroka Let's keep only entries that were created by us, e.g., Nina, Geri, Nick, etc. If two entries are enough for testing, we can use the file I've put there. If more that two entries are needed, let's add something entered by our people. I'm worried about having other people's data in our tests.